circle.yml

```diff
@@ -0,0 +1,12 @@
+machine:
+    services:
+        - docker
+
+dependencies:
+    override:
+        - /bin/true
```

Comment:
otherwise circle infers that it should `python setup.py install` on the host, which we don't want (all our tests happen in dockers).

I tried several ways like `override: []` or a list with just an empty string, but this is the only way I found that actually kills the default behavior :\